docker/docker-bake.hcl

```diff
@@ -21,7 +21,7 @@ variable "BASE_BUILDER_IMAGE_HASH" {
 # keep this in sync with the version in pyproject.toml
```

Comment:
This comment feels a bis misleading / outdated? because the poetry version is not directly referenced in `pyproject.toml`.

Reply:
That is true, I will update the comment